### ISSUE TITLE
Delay between transmit() and response 

### ISSUE THREAD

**Issue by cuihtlauac** [GPSecretariat/WebApis-for-se#8]
According to the relevant normative references, what is the maximum delay allowed to take place between a call to `Channel.transmit()` and receiving the corresponding `SEResponse`?

**Comment by serianox** [comment]
There is no limit to how long a command may last. A card can send as many _waiting time extension_ as it need to complete the execution of a command.

Existing API have a built-in timeout mechanism to prevent e.g. denial of service (see [     android.nfc.tech.IsoDep.setTimeout()](https://developer.android.com/reference/android/nfc/tech/IsoDep.html#setTimeout%28int%29)).

It is the responsibility of the application who knows beforehand how long a command may take up to to complete to configure the timeout mechanism. On the other hand, the default timeout should be set to a reasonable value (maybe between 200ms and 500ms).

A method or an attribute to configure the timeout should be added to the `Channel` interface.

``` webidl
interface Channel {
    readonly        attribute Session     session;
    readonly        attribute ChannelType channelType;
    readonly        attribute SEResponse? openResponse;
                    attribute int         timeout;
    Promise<SEResponse> selectNext ();
    Promise<SEResponse> transmit (SECommand cmd);
    Promise<Uint8Array> transmit (Uint8Array cmd);
    Promise<void>       close ();
};
```

`timeout` of type `int`
&nbsp;&nbsp;&nbsp;&nbsp;The timeout in ms of commands sent through this channel.

**Comment by opoto** [comment]
Fixed by commit 961d2c7e2df5ba361462b614b87dc1724f19b9c4
